src/variants/R9M_stock_ldscript.ld

```diff
@@ -0,0 +1,188 @@
+/*
```

Comment:
Should I try to get rid of that one?

Reply:
Ofc you can do that if you want.
I think it would be separate task anyway to cleanup all targets. Lets keep this PRs more focused just to BL additions.

Reply:
ok!